pkgs/development/tools/database/pyrseas/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, python27Packages, fetchFromGitHub }:
+
+let
```

Comment:
how about using a `with pythonPackages` on the top? It would remove a lot of noise.